Assets/Modules/Varietas Delectat/XML/UnitArt/VD_More_CIV4ArtDefines_Unit.xml

```diff
@@ -0,0 +1,2028 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- edited with XMLSPY v2004 rel. 2 U (http://www.xmlspy.com) by Firaxis Games (Firaxis Games) -->
+<!-- Sid Meier's Civilization 4 -->
+<!-- Copyright Firaxis Games 2005 -->
+<!-- -->
+<!-- Unit art path information -->
+<Civ4ArtDefines xmlns="x-schema:CIV4ArtDefinesSchema.xml">
+	<UnitArtInfos>
+<!--________________________________________________________________________________________________________________________________________________-->
+<!--                                                                                                                                                -->
+<!-- VD mod BEGIN Iranian UnitArt                                                                                                                   -->
+<!--________________________________________________________________________________________________________________________________________________-->
+		
+		<UnitArtInfo>
+			<Type>ART_DEF_UNIT_AXEMAN_IRANIAN</Type>
+			<Button>,Art/Interface/Buttons/Units/Warrior.dds,Art/Interface/Buttons/mem_units_atlas.dds,7,2</Button>
```

Comment:
All of these units use custom buttons from the `mem_units_atlas.dds` file, however usually VD unit varieties do not use different buttons. They should be changed to the default art and the atlas file should be deleted.

Reply:
I see. I just copy-pasted since I was a bit of in a rush. I'll work on it :)